src/components/Header.js

```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components';
+import logo from '../img/logo.png';
+import searchIcon from '../img/search.png';
+import cartIcon from '../img/shopping-cart.png';
+
+const StyledHeader = styled.header`
+  display: flex;
+  align-items: center;
+  border-bottom: 1px solid lightgray;
+`;
+
+const ImgRounded = styled.img`
+  border-radius: 50%;
+  width: 50px;
+`;
+
+const Title = styled.div`
+  padding: 1rem;
+  flex-grow: 2;
+  text-align: center;
+  font-size: 2rem;
+  font-weight: bolder;
+`;
```

Comment:
The styled components could also live in their own file, a folder for each component could be created to keep the things relevant to the component in one place, the styles, tests, definition, etc

For example:
- Header
  - `Header.component.jsx`
  - `Header.styles.jsx`
  - `Header.test.jsx`
  - `index.js` here you define a default export that contains what is inside the `Header.component.jsx` this way you can import the header like import Header from `'../components/Header/'`

By moving the styled components to their file, the component definition is kept clean and also, if you have a naming convention for all the files that contain styles, for example `<COMPONENT>.styles.jsx`, you can exclude the styles from the test coverage